docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,104 @@
+This guide was developed using a `Nokia G-240W-B` router. It should work for other models in the `G` series.
+
+### Device information
+
+- Device name: `Nokia G-240W-B`
+- Vendor: `Nokia`
+- Hardware version: `3FE56756ADBA`
+- Software version: `3FE47969AGCA21`
+- Chipset: `BCM6838`
+
+> Note:
+There is no single way to do it right. Chose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- The `Raspberry Pi` is accessible from the `LAN` network via `SSH`.
+- The `Raspberry Pi` is using static IP address, `IPv4` and `IPv6` as well.
+- The `ISP` provides `IPv6` address, prefix and gateway to the router.
+
+For this example, we will be using the `192.168.1.10` and `fe80::192:168:1:10` addresses for the following steps. You will need to substitute your own addresses in the following steps.
```

Comment:
I think this part can be deleted. If users set static addresses, they should know them. If not, you provide a section below how to find them.

Reply:
Sure thing, non used info must be removed; including the redundant text.